canonical-web-and-design/jaas.ai
# handle redirects

## Done

- add `canonicalwebteam.yaml_responses`
- add all redirects (from docs jujuchamrs)
- Use temporary index page that has updated links https://discourse.jujucharms.com/t/tmp-new-juju-documention-index/1868 (original index page will be updated once we decomission the other docs)

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029/docs
- play around docs.
- Make sure links don't have /t/blabla but a clean url
- Test the redirects in the redirects.yam and the deleted.yaml
